dtls/options.go

```diff
@@ -104,20 +104,36 @@ func WithGoPool(goPool GoPoolFunc) GoPoolOpt {
 
 // KeepAliveOpt keepalive option.
 type KeepAliveOpt struct {
-	keepalive *keepalive.KeepAlive
+	limitFails uint32
+	duration   time.Duration
```

Comment:
Same as in kit, probably it belongs here:
new config for coapgw proposed.

```
      keepalive:
        enabled: true
        timeout: 5s
        maxRetries: 6
```

could you please already reflect it in this MR ?

Reply:
Done